packages/create-plugin/templates/scenes-app/provisioning/plugins/app.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: 1
+
+apps:
+  - type: myorg-scenes-app
```

Comment:
Let's use templating for the type: `type: '{{ normalize_id pluginName orgName 'app' }}'`)